warehouses/fase/ingest/clone_postgresql_table.py

```
@@ -0,0 +1,95 @@
+            }
+            dummy_row[col_name] = None
+
+        def make_resource(t_name, j_cols, hints, d_row):
+            @dlt.resource(
+                name=t_name,
+                write_disposition="replace",
+                columns=hints,
+                table_format="iceberg"
+            )
+            def load_resource():
+                query = f'SELECT * FROM "{schema}"."{t_name}"'
```

Ruff (0.15.15)

[error] Possible SQL injection vector through string-based query construction

(S608)